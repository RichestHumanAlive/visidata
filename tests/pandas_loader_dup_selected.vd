sheet	col	row	longname	input	keystrokes	comment
global		filetype	set-option	pandas		
			open-file	sample_data/benchmarks.csv	o	
benchmarks		0	select-row		s	select current row
benchmarks		1	select-row		s	select current row
benchmarks		2	select-row		s	select current row
benchmarks			dup-selected		"	open duplicate sheet with only selected rows
