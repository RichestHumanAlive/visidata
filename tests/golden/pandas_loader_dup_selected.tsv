# comment
Total
189.05
999.50
